### DISCUSSION TITLE
id:pagewiz-takeover error

### DISCUSSION THREAD

**Discussion by ST0new** [projectdiscovery/nuclei-templates#4287] [Q&A] [answered]
Through the scan results found vulnerabilities, verification found that the domain name binding is not repeatable, can not achieve sub-domain takeover by binding the domain name
![image](https://user-images.githubusercontent.com/45659593/166173891-9b0ab348-7e98-4073-8a3c-f1030adc8c4c.png)
![image](https://user-images.githubusercontent.com/45659593/166173951-66fb87da-d45c-4ccd-ac4f-6b7dde7a28a9.png)

**Comment by olearycrew** [comment, marked as answer]
It's possible this was a false positive. If anyone sees this consistently, please report it through the False Positive issue reporter.
